ddev/src/ddev/utils/scripts/ci_matrix.py

```diff
@@ -49,6 +49,7 @@
   | metadata\.csv
   | pyproject\.toml
   | snmp/data/default_profiles/.+
+  | data/metrics.yaml
```

Comment:
```suggestion
  | .+/datadog_checks/.+/data/metrics\.yaml
```

Reply:
That turned out not to match, since what we match against looks like `datadog_checks/tomcat/data/metrics.yaml`, because we match against `remaining_path` from https://github.com/DataDog/integrations-core/blob/ab0c8533252a0af2a24c9bc6911311c06dba6376/ddev/src/ddev/utils/scripts/ci_matrix.py#L148